.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 npm-debug.log
 yarn-error.log
 yarn.lock
+.vscode
```

Comment:
Would you consider adding this to your global `.gitignore` instead on the repo? Or do you think it's good idea to have it here for for everyone?